### ISSUE TITLE
Use compiler to parent a "Placement Helper" to any of the "Half Wall Blocks"

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#1769]
![20170324123237_1](https://cloud.githubusercontent.com/assets/12244376/24301487/7f7bea74-108e-11e7-9137-bef4670a26ee.jpg)

**Comment by Luke18033** [comment]
What would probably be better is to have an 8-direction handle. That would not only solve this problem, but also allow more positions for placement helpers.

**Comment by LautaroL20** [comment]
Ill test if the placement model doesn´t go crazy when using the 8-directions handle. If its the case, ill have to re-arrange it in blender.

**Comment by LautaroL20** [comment]
Unfortunately, the Editor Model goes nuts when using the 8 positions feature.... See this:
![20170324130009_1](https://cloud.githubusercontent.com/assets/12244376/24302658/22c8c3e8-1092-11e7-8bcc-2a873790d44f.jpg)

**Comment by Luke18033** [comment]
The model would need to be repositioned for it to work.

**Comment by LautaroL20** [comment]
Alright, ive re-arranged the Editor Model and now it works like a charm, but still the instance goes the wrong way. Could you arrange the instance? (Im not much experienced in Hammer). Screenshots:

Model:
![20170324132809_1](https://cloud.githubusercontent.com/assets/12244376/24303859/41e0879e-1096-11e7-918a-02898d990a6a.jpg)

Instance:
![20170324133343_1](https://cloud.githubusercontent.com/assets/12244376/24303975/a6dde81c-1096-11e7-8bd8-dc1c578620e5.jpg)

**Comment by Luke18033** [comment]
> Could you arrange the instance?

I could, but currently I'm working on the Large Glass Panels and any commits that I make would go to the `glass_pan` branch.

**Comment by LautaroL20** [comment]
Yay, implemented for next release, heh? What do you say:
![20170324140200_1](https://cloud.githubusercontent.com/assets/12244376/24305220/f9010120-109a-11e7-82f8-53012812d569.jpg)

**Comment by Luke18033** [comment]
Technically it's not implemented yet, you haven't committed it.

**Comment by LautaroL20** [comment]
I know....Just a moment.

**Comment by Luke18033** [comment]
The only issue is that this will mess up rotation on old maps. Shouldn't be a problem though.

**Comment by LautaroL20** [comment]
I guess that every positive change worths it.

**Comment by Luke18033** [comment]
I've been thinking about how this would work, and I realized that we'll want to have a "start reversed" checkbox to make portals face the other way.

**Comment by LautaroL20** [comment]
Yes. When going trhough the center position, Screen shows correct, but at the moment of going trhough the Right-Left Portal positions, the Screen gets inverted in a 180° degrees. That option would fix this issue though.

**Comment by LautaroL20** [comment]
Start Reversed feature is now implemented. Do you know what does the "Start Enabled" option do? I readed the description but still i dont get it...

I will have to edit the item´s description aswell.

**Comment by Luke18033** [comment]
Start Enabled forces portals to match the orientation of the placement helper.

**Comment by LautaroL20** [comment]
Well, i hope i didn´t broken it with the implementation of all this new features. When you have the time, test the item and notify me of any issues. Item´s description now changed: 
![placement helper](https://cloud.githubusercontent.com/assets/12244376/24312043/8df9d724-10b5-11e7-9df3-d991eb97cf7c.png)

**Comment by libbybapa** [comment]
The option to uncheck 'Start enabled' is for floor/ceiling portals where it is desirable to have the portal center on the helper location but still allow the angle to change based on the position of player.

**Comment by Endermage77** [comment]
What is that round yellow portal in the upper right corner?
I didn't even know portals could do that.

**Comment by hop193** [comment]
Ir's the Co-op portal.

**Comment by Endermage77** [comment]
Yeah, but why does it look circular instead of the normal oval?

**Comment by Byzarru** [comment]
The portal was resized with one of the commands. I don't know which command, but `portals_resizeall` changes both portals.

**Comment by Endermage77** [comment]
That is actually a thing?

Let's make it a thing in BEE2!
For... uh... 
I dunno.

**Comment by Luke18033** [comment]
None of the portals were resized, the camera angle makes the yellow portal look bigger.

**Comment by hop193** [comment]
ksritthnl;dfnbl.fdjmgnb

**Comment by Byzarru** [comment]
The yellow portal is still resized, it takes up only half a panel. It would be much taller if it weren't.
